src/ekore/anomalous_dimensions/unpolarized/space_like/as2.py

```diff
@@ -10,12 +10,13 @@
 import numpy as np
 
 from eko import constants
-from .... import harmonics
-from ....harmonics.constants import log2, zeta2, zeta3
+from eko.constants import log2, zeta2, zeta3
+
+from ....harmonics import cache as c
 
 
 @nb.njit(cache=True)
-def gamma_nsm(n, nf, sx):
+def gamma_nsm(n, nf, cache, is_singlet):
```

Comment:
I think `is_singlet` should not appear as argument in any function except for `nsp` which can be used as singlet and as non singlet. 
What I mean is that calling this function with `is_singlet=True` is wrong, so we should not have this possibility. 
Viceversa hold for `gg, gq, qg, ps ...` 
@niclaurenti is the expert for the mixed QED QCD one.

Reply:
I agree with @giacomomagni . In this case you don't need to pass `is_singlet` because you already know that we are not in the singlet sector. It is better to call `get` with `is_singlet=False` (however notice  that `is_singlet` is useless for the first harmonic sums so consider using a default argument like `is_singlet=None`)